build/crds/edgecluster/edgecluster_v1.yaml

```diff
@@ -16,4 +16,28 @@ spec:
     - ec
   scope: Cluster
   subresources:
-    status: {}
\ No newline at end of file
+    status: {}
+  additionalPrinterColumns:
+    - name: LastHearbeat
+      type: date
+      JSONPath: .status.lastHeartBeat
+    - name: Healthy
+      type: boolean
+      description: Whether the cluster is healthy and accessible
+      JSONPath: .status.healthy
+    - name: nodes
+      type: string
+      description: nodes
+      JSONPath: .status.nodes
+    - name: edgenodes
```

Comment:
some names are in PascalCase and some are camelCase

Reply:
updated.